includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -211,6 +211,22 @@ public function custom_button_selector() {
 		return $this->is_custom_button() ? '#wcpay-custom-button' : '';
 	}
 
+	/**
+	 * Gets the product total price.
+	 *
+	 * @param object $product WC_Product_* object.
+	 * @return integer Total price.
```

Comment:
I think this method returns double (or float) since later will be multiplied by `100` in `WC_Payments_Utils::prepare_amount`

Reply:
Good catch, it actually is `mixed`. Sometimes it's an `integer`, sometimes a `double` and sometimes a `string` after doing `gettype( $product_price )` and looking at the source for `$product->get_price()`.

Fixed in 3c8e4be3744f8384e83c51f719252fbe6e0784d6